collect_app/src/main/java/org/odk/collect/android/upload/AutoSendWorker.java

```diff
@@ -301,13 +276,9 @@ private boolean atLeastOneFormSpecifiesAutoSend() {
     }
 
     private String formatOverallResultMessage(Map<String, String> resultMessagesByInstanceId) {
-        String message;
+        String message = "";
 
-        if (resultMessagesByInstanceId == null) {
-            message = resultMessage != null
-                    ? resultMessage
-                    : Collect.getInstance().getString(R.string.odk_auth_auth_fail);
-        } else {
+        if (resultMessagesByInstanceId != null) {
```

Comment:
What made `R.string.odk_auth_auth_fail` go away?

Reply:
The `resultMessage` field used to exist for non-form-specific failures which were considered fatal. If that notification method was called with a null messages map, it was assumed that an auth error happened (which I don't think was a good assumption).

Now, auth errors are not considered fatal -- there's just a message shown for the form that it happened for. Any other irrecoverable non-form-specific failures like not having a Google user set when using the gsheets "protocol" results in a message being passed directly to the notification message instead of using a field.